docs/guides/developing-components/getting-started.md

```diff
@@ -1,4 +1,4 @@
-# Developing Components >> Getting Started ||10
+# Developing Components: Getting Started
```

Comment:
If you could revert this change, we'd be happy to accept the changes below.

Reply:
Hi, Westbrook! I found the highlighted change on line 1 by clicking '**Edit this page on GitHub**' on the bottom of the [page](https://open-wc.org/guides/developing-components/getting-started/), but not in the version of the document that I changed on Jan 20. I'm afraid to create an unnecessary pull request trying to get the first one accepted.

Reply:
```suggestion
# Developing Components >> Getting Started ||10
```